src/components/admin/UserProfileModal/components/ProfileModalSectionHeader.tsx

```diff
@@ -0,0 +1,9 @@
+export interface ProfileModalSectionHeaderProps {
```

Comment:
If this interface isn't used outside this file, it will be shorter `ts-prune.result.txt` after running `npm run prune` to just 
```suggestion
interface ProfileModalSectionHeaderProps {
```

This comment should be the same for all the other `export interface` or in general `export` cases.

Reply:
done: https://github.com/sparkletown/sparkle/pull/2931/commits/fa80b08ff39c2c3413f9336652f0ebbaedeb1398